core/lib/config/src/configs/snapshot_recovery.rs

```diff
@@ -0,0 +1,41 @@
+use std::num::NonZeroUsize;
+
+use serde::Deserialize;
+use zksync_basic_types::L1BatchNumber;
+
+#[derive(Debug, Clone, PartialEq, Deserialize, Default)]
+pub struct Tree {
```

Comment:
Bikeshedding: IMO, it would make sense to name these structs `TreeRecoveryConfig` / `PostgresRecoveryConfig`, so that they are less ambiguous if used in isolation.